.pipelines/android_packaging.yml

```diff
@@ -19,57 +18,46 @@ jobs:
       - script: brew install coreutils ninja
         displayName: Install coreutils and ninja
 
-      - script: |
-          mkdir cmake/external
-          cd cmake/external
-          git clone https://github.com/microsoft/onnxruntime.git
-          cd onnxruntime
-          git checkout v1.12.1
-        displayName: Download Onnxruntime
-
-      - script: |
-          python tools/python/run_android_emulator.py \
-            --android-sdk-root ${ANDROID_SDK_ROOT} \
-            --create-avd --system-image "system-images;android-30;google_apis;x86_64" \
-            --start --emulator-extra-args="-partition-size 4096" \
-            --emulator-pid-file $(Build.BinariesDirectory)/emulator.pid
-        displayName: Start Android emulator
-        workingDirectory: $(OnnxruntimeDirectory)
-
-      - task: JavaToolInstaller@0
-        displayName: Use jdk 11
-        inputs:
-          versionSpec: "11"
-          jdkArchitectureOption: "x64"
-          jdkSourceOption: "PreInstalled"
-
       - script: |
           python ./tools/gen_selectedops.py ./tools/android/package_ops.config
         displayName: "Generate selected ops CMake file"
 
-      - bash: |
-          set -e -x
+      - template: templates/with-running-android-emulator-steps.yml
+        parameters:
+          steps:
 
-          python ./tools/android/build_aar.py \
-            --output-dir $(Build.BinariesDirectory)/android_aar \
-            --config $(buildConfig) \
-            --cmake-extra-defines OCOS_ENABLE_SELECTED_OPLIST=ON
+          # Start switching to jdk 11 after the Android Emulator is started
+          # since Android SDK manager requires java 8
```

Comment:
Does the Android SDK manager still require an ancient version of Java?

https://stackoverflow.com/a/65782803 suggests maybe not.

Reply:
it works with a newer version. thanks for the suggestion